XRTK-Core/Packages/com.xrtk.core/Interfaces/InputSystem/Handlers/IMixedRealityHandMeshHandler.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine.EventSystems;
+using UnityEngine;
+using XRTK.EventDatum.Input;
+
+namespace XRTK.Interfaces.InputSystem.Handlers
+{
+    /// <summary>
+    /// Interface to implement for hand mesh information.
+    /// </summary>
+    public interface IMixedRealityHandMeshHandler : IEventSystemHandler
+    {
+        void OnHandMeshUpdated(InputEventData<HandMeshInfo> eventData);
+    }
+
+    // See BaseHandVisualizer.OnHandMeshUpdated for an example of how to use the
+    // hand mesh info to render a mesh.
+    public class HandMeshInfo
```

Comment:
I recommend putting this class in its own file.

I also recommend adding Handedness to this class